backend/src/packages/chats/chat.controller.ts

```diff
@@ -148,11 +209,99 @@ class ChatController extends BaseController {
     return {
       status: HTTPCode.CREATED,
       payload: await this.chatService.create({
-        members: [options.user.id],
         chatEntity,
+        userId: options.user.id,
+        message: options.body.message,
       }),
     };
   }
+
+  /**
+   * @swagger
+   * /chats/{id}/messages:
+   *   post:
+   *     description: Create a new chat message
+   *     parameters:
+   *       -  in: path
+   *          description: Chat id
+   *          name: id
+   *          required: true
+   *          type: number
+   *          minimum: 1
+   *     requestBody:
+   *       description: Create message data
+   *       required: true
+   *       content:
+   *         application/json:
+   *           schema:
+   *             type: object
+   *             properties:
+   *               message:
+   *                 type: string
+   *     responses:
+   *       201:
+   *         description: Successful operation
+   *         content:
+   *           application/json:
+   *             schema:
+   *               $ref: '#/components/schemas/ChatMessage'
+   */
+  private async createMessage(
+    options: APIHandlerOptions<{
+      body: ChatMessageCreateRequestDto;
+      params: { id: string };
+      user: UserAuthResponseDto;
+    }>,
+  ): Promise<APIHandlerResponse> {
+    const chatMessageToCreateData: ChatMessageCreatePayload = {
+      chatId: Number(options.params.id),
+      message: options.body.message,
+      senderId: options.user.id,
+    };
+
+    return {
+      status: HTTPCode.CREATED,
+      payload: await this.chatService.createMessage(chatMessageToCreateData),
+    };
+  }
+
+  /**
+   * @swagger
+   * /chats/{id}/messages:
+   *   get:
+   *     description: Returns all chat messages
+   *     parameters:
+   *       -  in: path
+   *          description: Chat id
+   *          name: id
+   *          required: true
+   *          type: number
+   *          minimum: 1
+   *     responses:
+   *       200:
+   *         description: Successful operation
+   *         content:
+   *           application/json:
+   *             schema:
+   *               type: object
+   *               properties:
+   *                 items:
+   *                   type: array
+   *                   items:
+   *                     $ref: '#/components/schemas/ChatMessage'
+   */
+  private async findAllMessagesByChatId(
+    options: APIHandlerOptions<{
+      params: { id: string };
```

Comment:
Maybe make separate type for it cause you use it multiple times, for instance ChatParamsDto

Reply:
actually it was a type, but @Liza-Veis suggested to remove the type here https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/301#discussion_r1330033506

Reply:
yes, we will leave it this way